exchange/exchange.go

```diff
@@ -79,6 +81,24 @@ type bidResponseWrapper struct {
 	bidder       openrtb_ext.BidderName
 }
 
+type UUIDGenerator interface {
+	NewUUID() string
+	Enabled() bool
+}
+
+type bidIDGenerator struct {
+	enabled bool
+}
+
+func (big *bidIDGenerator) Enabled() bool {
+	return big.enabled
+}
+
+func (big *bidIDGenerator) NewUUID() string {
+	rawUuid, _ := uuid.NewV4()
```

Comment:
Please don't ignore errors. If it's possible to have an error from `uuid.NewV4()`, we'll need to bubble that up the call chain.

Reply:
Yes, added